contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1652,6 +1653,17 @@ define_escape_hatch_variables(void)
 							 PGC_USERSET,
 							 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 							 NULL, NULL, NULL);
+	
+	/* INLINE option */
+	DefineCustomEnumVariable("babelfishpg_tsql.escape_hatch_inline_function",
+							 gettext_noop("escape hatch for INLINE = {ON|OFF} option in CREATE FUNCTION"),
+							 NULL,
+							 &escape_hatch_inline_function,
+							 EH_STRICT,
```

Comment:
Can you check with Rob if default should really needs to be STRICT?

Reply:
Yes, already checked with him. Since, we don't support this option, we should throw an error.